InnerEye/ML/SSL/lightning_containers/ssl_container.py

```diff
@@ -147,22 +147,28 @@ def create_model(self) -> LightningModule:
         # For small images like CIFAR, if using a resnet encoder, switch the first conv layer to a 3x3 kernel instead
         # of a 7x7 conv layer.
         use_7x7_first_conv_in_resnet = False if self.ssl_training_dataset_name.value.startswith("CIFAR") else True
+
+        # Adjust the learning rate to work with multi gpu
```

Comment:
Should we add a comment explaining why we would want to do this? Maybe a reference to the FB paper too?